src/main/java/com/rocketden/main/util/Color.java

```diff
@@ -0,0 +1,29 @@
+package com.rocketden.main.util;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@EqualsAndHashCode
+public class Color {
+
+    private static final String INVALID_COLOR_NUM =
+        "The r, g, and b fields must all be between 0 and 255.";
+
+    public Color(int r, int g, int b) {
```

Comment:
This looks great; however what are your thoughts on simply using strings such as `#0099FF`? My only thinking behind this is that it might be easier to integrate those with styled-components (a component can accept a color string prop and we can assign it by simply using `background-color: ${({ color }) => color};`

However since we already have this Color class I can understand if we want to keep it this way too - I'm sure we'll be able to figure out a way to parse it on the frontend

Reply:
I think that'd make sense (I didn't do this initially because I thought it wouldn't work for some reason, but I was wrong) -- I'll go ahead and change this after reviewing all of these PRs.

Reply:
@alankbi If I switch the `Color` object to use a String, were you saying that I really don't need a `Color` class in the first place? Should I just replace that with a String?

Reply:
I'm not sure - maybe it could still be used to hold all the different colors? I don't have too much of a preference on that though

Reply:
Ok, I'll just keep the Color class because perhaps it could be expanded in the future to include other symbols or whatnot.